app/controllers/placements.js

```diff
@@ -505,6 +508,8 @@ exports.new_placement_subject_post = (req, res) => {
       },
       errors
     })
+  } else if (!mentors.length) {
+    res.redirect(`/organisations/${req.params.organisationId}/placements/new/check`)
   } else {
     res.redirect(`/organisations/${req.params.organisationId}/placements/new/mentor`)
   }
```

Comment:
The first if/else is to do with errors. The second if/else is to check if there are mentors. Whilst it works, to make the code a little easier to read, perhaps put the second if statement inside the first else:

```suggestion
} else {
    if (!mentors.length) {
      res.redirect(`/organisations/${req.params.organisationId}/placements/new/check`)
    } else {
      res.redirect(`/organisations/${req.params.organisationId}/placements/new/mentor`)
    }
  }
```